condor.go

```diff
@@ -97,15 +96,7 @@ func New(c *viper.Viper, client Messenger, fs fsys, condorSubmit, condorRm strin
 	}
 }
 
-func (cl *CondorLauncher) storeConfig(s *model.Job) (string, error) {
-	uselimit := len(s.Inputs()) + 2 // 2 comes from 1 for writing, one for the output job.
```

Comment:
Yeah. Interestingly I think the calculation here was wrong anyway, since at some point we changed everything to use an input list rather than separate containers per input. But many/most jobs used one input anyway, so it usually wasn't different.